ondemand/Dockerfile

```diff
@@ -35,4 +35,6 @@ COPY notebook_data /data/notebook_data
 COPY initializers/dashboard.rb /etc/ood/config/apps/dashboard/initializers/dashboard.rb
 COPY themes/hpc-coop /usr/share/ondemand-dex/web/themes/hpc-coop
 
+RUN sed -i '/  passenger_load_shell_envvars off;/a   passenger_preload_bundler on;' /opt/ood/nginx_stage/templates/pun.conf.erb
```

Comment:
This file doesn't seem to exist for the arch64 builds. Does this file exists somewhere else for the source install? If so we'll need to move this to the arch specific sections of the Docker file (similar to how we patch the vnc.rb). If not, perhaps we're missing something?

```
[root@3efdeca69902 /]# uname -m
x86_64
[root@3efdeca69902 /]# ls /opt/ood/
VERSION  mod_ood_proxy	ondemand	     ood_auth_map
hooks	nginx_stage	ood-portal-generator
```

```
[root@e007ed863a1f ~]# uname -m
aarch64
[root@e007ed863a1f ~]# ls /opt/ood/
ondemand
```

Reply:
What? That doesn't seem right at all...

Reply:
Here's where we move that stuff in the arm script.

https://github.com/ubccr/hpc-toolset-tutorial/blob/2930c929f401fbe07ff34971dedb5d7527126322/ondemand/install-ood-arm64.sh#L47

Reply:
Nevermind! My bad, was looking at an old container image. It's all there: 

```
[root@bcd5419b066e /]# uname -m
aarch64
[root@bcd5419b066e /]# ls /opt/ood/
mod_ood_proxy  nginx_stage  ondemand  ood-portal-generator  ood_auth_map
```